### PR TITLE
feat/qb2194: relay missed tx

### PR DESCRIPTION
**Pull request by bjlqsnetwork**
- qb2194: relay missed tx from stchain to sp
- qb2201: fix unnecessary error message during transferring slice

### CODE REVIEW

File: pp/namespace/api.go [hunk 1]
```diff
@@ -98,6 +105,17 @@ func Apis() []rpc.API {
 	}
 }
 
+func RelayApis() []rpc.API {
```
Comment (alexstratos) [inline pp/namespace/api.go:108]: not belong to here

File: relay/stratoschain/grpc/query.go [hunk 2]
```diff
@@ -100,3 +104,39 @@ func QueryMetaNode(p2pAddress string) (err error) {
...
+	}
+	// process relayed events
+	events := handlers.ProcessEvents(*resp.TxResponse)
+	for msgType, event := range events {
+		if handler, ok := handlers.Handlers[msgType]; ok {
+			go handler(event)
+		} else {
+			utils.ErrorLogf("No handler for event type [%v]", msgType)
+		}
+	}
```
Comment (alexstratos) [inline relay/stratoschain/grpc/query.go:140]: shouldn't be part of logic of this function

Comment (bjlqsnetwork) [reply relay/stratoschain/grpc/query.go]: solved

File: pp/namespace/api.go (hunk 1 above)
Comment (bjlqsnetwork) [reply pp/namespace/api.go]: removed

**Review by alexstratos — APPROVED** [review]

**Pull request merged**